tsconfigdoc.json

```diff
@@ -271,6 +274,9 @@
     {
       "path": "./packages/vdom-extension"
     },
+    {
+      "path": "./packages/vega4-extension"
```

Comment:
I do not understand this addition.There does not seem to be `vega4-extension` or `tabmanager-extension` on master, but I may be wrong.

Reply:
That was automatically added by tooling. I can remove it

Reply:
Do you maybe have these extra (untracked) packages locally?

Reply:
It seems both packages were part of the codebase but were removed at some point, see https://github.com/jupyterlab/jupyterlab/commit/9bf678ac525d9372b9371e279a4cebf8c8da1cdf or https://github.com/jupyterlab/jupyterlab/commit/d002809e0579b37ce45270b7b744e72798bf9f57 and that including them causes [documentation build](https://readthedocs.org/projects/jupyterlab/builds/13736313/) to fail:

```
$ typedoc --options typedoc.js
Error: [91merror[0m[90m TS6053: [0mFile '/home/docs/checkouts/readthedocs.org/user_builds/jupyterlab/checkouts/10160/packages/tabmanager-extension' not found.

Error: [91merror[0m[90m TS6053: [0mFile '/home/docs/checkouts/readthedocs.org/user_builds/jupyterlab/checkouts/10160/packages/vega4-extension' not found.

error Command failed with exit code 3.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
These could indeed automatically by the tooling have been added if the (empty) folder was still on disk.